### PR TITLE
Benchmark: memtable, bufferwrite, overflow

### PR DESCRIPTION
**Pull request by liukun4515**
Add some bench for write.

### CODE REVIEW

File: iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/overflow/ioV2/OverflowProcessorBenchmark.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        Map<String,Object> parameters = new HashMap<>();
+        parameters.put(FileNodeConstants.OVERFLOW_FLUSH_ACTION, new Action() {
+            @Override
+            public void act() throws Exception {
+                System.out.println(FileNodeConstants.OVERFLOW_FLUSH_ACTION);
+            }
+        });
+        parameters.put(FileNodeConstants.FILENODE_PROCESSOR_FLUSH_ACTION, new Action() {
+            @Override
+            public void act() throws Exception {
```
Comment (jixuan1989) [inline iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/overflow/ioV2/OverflowProcessorBenchmark.java]: in OverflowProcessor.java, `flushFileNodeProcessorAction` will write data on disk and call a `sync`.  Simplify it here is suitable?

**Comment by jixuan1989** [comment]
@liuruiyiyang  see, FileNodeManagerBenchmark, what is the difference with your benchmark scenario?

File: iotdb/src/test/java/cn/edu/tsinghua/iotdb/qp/bench/QueryParseBenchmark.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
...85434416303107,787.74,728.2850866795781,33960.24564285714,6.651756751280603,33960.24564285714,38.51408765504662,1160.4112366716558,0.0,33960.24564285714,33960.24564285714,6.651756751280603,33960.24564285714,787.74,6.651756751280603,0.0,6.651756751280603,33960.24564285714,787.74,787.74,656.2029144236096,593.309858647129,787.74,0.0,787.74,0.0,33960.24564285714,33960.24564285714,1127.350328754438,33960.24564285714,33960.24564285714,33960.24564285714,787.74,1180.971235067114,661.7070251519021,33960.24564285714,0.0,0.0,1205.10868101537,792.940812264317,0.0,573.1607619219858,6.651756751280603,156.47709039629825,33960.24564285714,809.2345221425896,0.0,6.651756751280603,787.74,6.651756751280603,33960.24564285714,33960.24564285714,965.7042945108797,787.74,6.651756751280603,6.651756751280603,787.74,6.651756751280603,0.0,6.651756751280603,0.0,6.651756751280603,317.6444399747479,787.74,0.0,787.74,6.651756751280603,33960.24564285714,1028.5886481610971,107.99305540655205,0.0,33960.24564285714,787.74,252.59133785592041,725.8255243198603,443.50233343042555,787.74,6.651756751280603,6.651756751280603,33960.24564285714,33960.24564285714,33960.24564285714,787.74,33960.24564285714,441.68146442890685)";
```
[the remark this answers is not in the document]
Comment (liukun4515) [reply iotdb/src/test/java/cn/edu/tsinghua/iotdb/qp/bench/QueryParseBenchmark.java]: random SQL is better, but it isn't necessary for the SQL parse benchmark. (just insert SQL)

File: iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/overflow/ioV2/OverflowProcessorBenchmark.java (hunk 1 above)
Comment (liukun4515) [reply iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/overflow/ioV2/OverflowProcessorBenchmark.java]: overflow benchmark does not contain the flush time.
In the normal situation, the flush operation run in a background thread, it doesn't affect the speed of insert data.
**In the iotdb benchmark test, the speed of insert is low from the beginning.**

**Comment by jixuan1989** [comment]
I suggest that put all of these classes into /iotdb/benchmark  folder.

File: iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/bufferwrite/BufferWriteBenchmark.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    static {
+        for(int i = 0;i<numOfDevice;i++){
+            deviceIds[i] = String.valueOf("d"+i);
+        }
+    }
+    private static int numOfMeasurement = 100;
+    private static String[] measurementIds  = new String[numOfMeasurement];
+    private static FileSchema fileSchema = new FileSchema();
+    private static TSDataType tsDataType = TSDataType.INT64;
+    private static int numOfPoint = 1000;
```
[the remark this answers is not in the document]
Comment (jt2594838) [reply iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/bufferwrite/BufferWriteBenchmark.java]: And I notice that these codes are similar across the benchmarks. Maybe you can extract them with a base class Benchmark and a method initialize()?

File: iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/bufferwrite/BufferWriteBenchmark.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    static {
+        for(int i = 0;i< numOfMeasurement;i++){
+            measurementIds[i] = String.valueOf("m"+i);
```
Comment (jt2594838) [inline iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/bufferwrite/BufferWriteBenchmark.java]: I wonder if the call of String.valueOf() is necessary.

Comment (liukun4515) [reply iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/bufferwrite/BufferWriteBenchmark.java]: > I wonder if the call of String.valueOf() is necessary.

no, we can just use `measurementIds[i] = i`

Comment (jixuan1989) [reply iotdb/src/test/java/cn/edu/tsinghua/iotdb/engine/bufferwrite/BufferWriteBenchmark.java]: measurementIds[i] = "m"+i;

**Pull request merged**